src/build-actions.js

```diff
@@ -82,7 +82,10 @@ const getWebpackConfig = async (actionPath, root, tempBuildDir, outBuildFilename
 
   // we have 1 required plugin to make sure is present
   config.plugins = config.plugins || []
-  config.plugins.push(new webpack.DefinePlugin({ WEBPACK_ACTION_BUILD: 'true' }))
+  config.plugins.push(new webpack.DefinePlugin({
+    WEBPACK_ACTION_BUILD: 'true',
+    'process.env.AIO_ENV_IS_STAGE': process.env.AIO_ENV_IS_STAGE || false
```

Comment:
we can define this similar to `WEBPACK_ACTION_BUILD ` instead of adding to process.env
`config.plugins.push(new webpack.DefinePlugin({ AIO_ENV_IS_STAGE: JSON.stringify(process.env.AIO_ENV_IS_STAGE)}))`

Reply:
Then we have to modify action code to look for it